resources/ajuna/paseo/ajuna-paseo-raw-unsorted.json

```diff
@@ -1,16 +1,19 @@
 {
   "name": "Ajuna Paseo",
-  "id": "ajuna-ajuna-westend",
+  "id": "ajuna-paseo",
```

Comment:
This change implies that we have to move the chain-db from `ajuna-ajuna-westend` to `ajuna-paseo` for the running system.